### DISCUSSION TITLE
Are the results stored in memory??

### DISCUSSION THREAD

**Discussion by YasaswiniSireddy** [deepdoctection/deepdoctection#364] [Q&A] [answered]
How can I use the "detect result list" of image layout out the layout.py. Are they stored some where in memory?? 
        detect_result_list = self.predictor.predict(np_image)
If they are stored in memory can you help me to know how can i use them in other files.

**Comment by JaMe76** [comment, marked as answer]
Raw results from predictors (e.g. 'DetectionResult') will be validated and converted into some derived `Annotation` instance. You can see a snippet in the code below: `self.dp_manager.set_image_annotation` will do the validation and the conversion into the target format.

https://github.com/deepdoctection/deepdoctection/blob/b975421d5fee393076eb02ab3d394cc385dd7736/deepdoctection/pipe/layout.py#L90-L98

Raw results from predictors are not being kept in memory.

**Reply by YasaswiniSireddy** [reply]
yeah okay @JaMe76. Thank you for clarification.
